From 57ee32eac82fccce10d18a8902f146ca20cca787 Mon Sep 17 00:00:00 2001
From: Vikram Pandita <vikram.pandita@ti.com>
Date: Thu, 12 Apr 2012 18:45:33 -0700
Subject: [PATCH 2/4] hack: disable audio logging noise

Signed-off-by: Vikram Pandita <vikram.pandita@ti.com>
---
 sound/soc/soc-pcm.c |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/sound/soc/soc-pcm.c b/sound/soc/soc-pcm.c
index 62e05a6..6df5796 100644
--- a/sound/soc/soc-pcm.c
+++ b/sound/soc/soc-pcm.c
@@ -1931,8 +1931,8 @@ int soc_dpcm_runtime_update(struct snd_soc_dapm_widget *widget)
 
 		paths = fe_path_get(fe, SNDRV_PCM_STREAM_PLAYBACK, &list);
 		if (paths < 0) {
-			dev_warn(fe->dev, "%s no valid %s route from source to sink\n",
-					fe->dai_link->name,  "playback");
+			//dev_warn(fe->dev, "%s no valid %s route from source to sink\n",
+			//		fe->dai_link->name,  "playback");
 			ret = paths;
 			goto out;
 		}
@@ -1960,8 +1960,8 @@ capture:
 
 		paths = fe_path_get(fe, SNDRV_PCM_STREAM_CAPTURE, &list);
 		if (paths < 0) {
-			dev_warn(fe->dev, "%s no valid %s route from source to sink\n",
-					fe->dai_link->name,  "capture");
+			//dev_warn(fe->dev, "%s no valid %s route from source to sink\n",
+			//		fe->dai_link->name,  "capture");
 			ret = paths;
 			goto out;
 		}
@@ -2336,8 +2336,8 @@ int soc_dpcm_fe_dai_open(struct snd_pcm_substream *fe_substream)
 	fe->dpcm[stream].runtime = fe_substream->runtime;
 
 	if (fe_path_get(fe, stream, &list) <= 0) {
-		dev_warn(fe->dev, "asoc: %s no valid %s route from source to sink\n",
-			fe->dai_link->name, stream ? "capture" : "playback");
+		//dev_warn(fe->dev, "asoc: %s no valid %s route from source to sink\n",
+		//	fe->dai_link->name, stream ? "capture" : "playback");
 			return -EINVAL;
 	}
 
-- 
1.7.4.1

